recipes/drwav/all/CMakeLists.txt

```
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.4)
+project(dr_wav C)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATHS)
```
Preserve correct behavior on Mac